Tests/ResilientDecodingTests/ResilientRawRepresentableDictionaryTests.swift

```diff
@@ -48,7 +51,10 @@ final class ResilientRawRepresentableDictionaryTests: XCTestCase {
     XCTAssertEqual(mock.$optionalResilientDictionaryOfFrozenType.errors.count, 0)
     #endif
   }
-
+  
+  /**
+   - note: We keep the non-optional properties in the JSON so they do not report errors
+   */
```

Comment:
Since this test name claims no errors should we add `expectedErrorCount: 0` to our decoding method?

Reply:
This is the default.